G1ANT.Addon.GoogleDocs/G1ANT.Addon.GoogleDocs.Tests/Commands/googlesheet.download/ShouldDownloadSheet_WhenOpenAndDownloadSheet.G1ANT

```diff
@@ -0,0 +1,13 @@
+﻿addon googledocs version 2.205.17304.1000
+addon core version 4.99.18340.1354
+addon language version 4.99.18340.1354
+
+♥FileID = ‴147EH2vEjGVtbzzkT6XaI0eNZlY5Ec91wlvxN3HC4GMc‴
```

Comment:
That might become a pain in the ass if someone remove the document and we will have to change it in multiple places. Unfortunately, since `task.include` does not support paths relative to the executing script (quite a big fuckup, isn't it? I'm trying to fix it in other task), I have no clue how to do that better.